### ISSUE TITLE
Use actual API credentials

### ISSUE THREAD

**Issue by yannbolliger** [yannbolliger/notion-exporter#2]
Maybe we can use the _official_ Notion API to get a token, then use the old workflow.
At least, one should be able to provide login credentials instead of the rather hacky `token_v2`.

**Comment by shahrukhalam** [comment]
I was not able to use the CLI as there was no `file_token` in cookies, I guess it is related to this issue, right?

**Comment by yannbolliger** [comment]
> I was not able to use the CLI as there was no `file_token` in cookies, I guess it is related to this issue, right?

@shahrukhalam  No, this was #3 and has been fixed with the latest release. Please also check the updated README.

**Comment by shahrukhalam** [comment]
> @shahrukhalam  No, this was #3 and has been fixed with the latest release. Please also check the updated README.

@yannbolliger But, I was not able to find it in the cookies 🤔
I have followed the ReadMe & I guess it didn't change related to how to get _file_token_

In one of the comments on Issue #3, there is a mention of url of zip file, is that the way? if yes, what's it?

**Comment by yannbolliger** [comment]
@shahrukhalam I just retried the [cookie explanation](https://github.com/yannbolliger/notion-exporter#how-to-retrieve-the-cookies) in Firefox. I see both cookies (`file_token`, `token_v2`) even if I log in with a fresh private mode tab. Maybe try that once and make sure to check for cookies in the cookies/storage tab of the developer tools – not on a single request.

**Comment by shahrukhalam** [comment]
@yannbolliger Ah, I was using Safari, now i am able to see them in Chrome
Thanks much for the help 💚

**Comment by kaedea** [comment]
Hey, @shahrukhalam 
Did you know is there an [offical Notion API](https://developers.notion.com/reference/intro) to export Notion Data or to get cookies token? I used `https://notion.so/api/v3/loginWithEmail` to retrieve cookies token ago, but it seems been banned by Notion right now.

**Comment by louislspn** [comment]
For those wondering why the `file_token` is not present in cookies : you need to export something at least one time from your browser in order to store this token.

After that, copy `file_token` and `token_v2` and it should work fine.

**Comment by yannbolliger** [comment]
> Did you know is there an [offical Notion API](https://developers.notion.com/reference/intro) to export Notion Data or to get cookies token? I used `https://notion.so/api/v3/loginWithEmail` to retrieve cookies token ago, but it seems been banned by Notion right now.

Yes, this is the original goal & intention of this issue, to use the official API to get tokens/credentials in an official way. This would also mean that you don't have to recreate tokens every 90 days.
If someone here is willing to help/provide a PR, you are more than welcome.

Also, thanks @louislspn for adding that explanation.
